src/tests/ftest/util/command_utils_base.py

```diff
@@ -302,6 +302,14 @@ def get_params(self, test):
         for name in self.get_param_names():
             getattr(self, name).get_yaml_value(name, test, self.namespace)
 
+    def update_params(self, **params):
+        """Update each of provided parameter name and value pairs."""
+        for name, value in params.items():
+            try:
+                getattr(self, name).update(value, name)
+            except AttributeError:
+                raise CommandFailure("Unknown parameter: {}".format(name))
```

Comment:
(pylint-raise-missing-from) Consider explicitly re-raising using the 'from' keyword